packages/bitcore-node/src/modules/ethereum/api/csp.ts

```diff
@@ -13,15 +13,24 @@ import { SpentHeightIndicators } from '../../../types/Coin';
 import { EthListTransactionsStream } from './transform';
 import { ERC20Abi } from '../abi/erc20';
 import { Transaction } from 'web3/eth/types';
-import { EventLog } from 'web3/types';
 import { partition } from '../../../utils/partition';
 import { WalletAddressStorage } from '../../../models/walletAddress';
-
+import { Contract } from 'web3-eth-contract';
+type ContractAbiInput = ConstructorParameters<typeof Contract>[0];
+interface EventLog {
+  event: string;
+  address: string;
+  returnValues: any;
+  logIndex: number;
+  transactionIndex: number;
+  transactionHash: string;
+  blockHash: string;
+  blockNumber: number;
+  raw?: {data: string; topics: any[]};
+}
 interface ERC20Transfer extends EventLog {
   returnValues: {
-    _from: string;
-    _to: string;
-    _value: string;
+    [key: string]: string;
```

Comment:
`EventData[]` returnValues typing is:

```
  [key: string]: any;
```

Not compatible with:
```
  _from: string;
  _to: string;	
  _value: string;
```

Reply:
https://github.com/ethereum/web3.js/blob/043cb81d40ede0f6ef1889222c5671f9678e409e/packages/web3-eth-contract/types/index.d.ts#L146

Reply:
```
interface ERC20Transfer extends EventLog<{
  _from: string;
  _to: string;	
  _value: string;
}>
```